.pipelines/ci.yml

```diff
@@ -74,6 +76,9 @@ stages:
           name: 1es-aro-ci-pool
         steps:
           - checkout: self
+          - task: Docker@2
+            inputs:
+              command: 'login'
```

Comment:
```suggestion
              command: 'login'
              containerRegistry: arointsvc #service connection name

Reply:
Does it need to be `command: login` instead of `command: 'login'`? Does it matter?

Docs have no single quotes: https://learn.microsoft.com/en-us/azure/devops/pipelines/tasks/reference/docker-v2?view=azure-pipelines&tabs=yaml

Reply:
It's just YAML, there is no difference between `'string'` and `string` unless the literal contains reserved characters